sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 ## 7.0.0-preview.6 (Unreleased)
 
+- Renames
```

Comment:
Renames probably warrant the Breaking Changes header like this:

https://github.com/Azure/azure-sdk-for-js/blob/6c25936d545fa021f42707583391be93378ce079/sdk/servicebus/service-bus/CHANGELOG.md#breaking-changes

Reply:
```suggestion
### Breaking Changes
```